libraries/AP_Proximity/AP_Proximity_Params.cpp

```diff
@@ -108,6 +108,14 @@ const AP_Param::GroupInfo AP_Proximity_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_MAX", 17, AP_Proximity_Params, max_m, 0.0f),
 
+    // @Param: _ADDR
+    // @DisplayName: Bus address of sensor
+    // @Description: This sets the bus address of the sensor, where applicable. Used for the I2C and DroneCAN sensors to allow for multiple sensors on different addresses.
+    // @Range: 0 127
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("_ADDR", 25, AP_Proximity_Params, address, 0),
```

Comment:
why 25?

Reply:
We can remove the "This sets the" part of the description because all parameters set something.  Let's just start with "The bus address of the sensor.."